### ISSUE TITLE
release event w/o a push event

### ISSUE THREAD

**Issue by absalom-muc** [hideakitai/Debouncer#4]
When a button is pushed for a time <debounce time and then released, then the release event is triggered. I believe there should be no release event without a push-event before. What do you think?

Thanks
absalom

**Comment by hideakitai** [comment]
Hi, could you clarify what you want with your example code?

**Comment by absalom-muc** [comment]
For demonstration I use your [simple example](https://github.com/hideakitai/Debouncer/tree/master/examples/simple), but increased debounce_duration_ms to 200ms (that makes it easier to reproduce). Please find the log:
```
07:38:42.292 -> fall : 0
07:38:42.292 -> changed : 0
07:38:42.292 -> onFall
07:38:42.292 -> onChanged
07:38:42.863 -> rise : 0
07:38:42.863 -> changed : 1
07:38:42.863 -> onRise
07:38:42.863 -> onChanged
07:39:08.863 -> rise : 1
07:39:08.863 -> changed : 2
07:39:08.863 -> onRise
07:39:08.863 -> onChanged
```
07:38 I pressed the button >200ms, so the resulting events are as expected
07:39 I pressed the button <200ms, the rise event was triggered

From my point of view the rise event at 07:39 is wrong. There should be no rise-event without preceding fall-event.

**Comment by hideakitai** [comment]
@absalom-muc Thanks, this is fixed in v0.2.9. Please try it, and if you still have the problem, please feel free to reopen it!

**Comment by absalom-muc** [comment]
It works like a charm - thank you @hideakitai. Appreciate your quick response.
